### ISSUE TITLE
Crash after upgrade to urwid 2.0

### ISSUE THREAD

**Issue by amanusk** [Nanoseb/ncTelegram#32]
When trying to open nctelegram after upgrade to urwid version 2.0, the program crashes
```
Traceback (most recent call last):
  File "/usr/local/bin/nctelegram", line 93, in <module>
    ncTelegram.Telegram_ui(config_full)
  File "/usr/local/lib/python3.5/dist-packages/ncTelegram/__init__.py", line 73, in __init__
    self.msg_widget = MessageWidget(self)
  File "/usr/local/lib/python3.5/dist-packages/ncTelegram/ui_msgwidget.py", line 25, in __init__
    self.get_history()
  File "/usr/local/lib/python3.5/dist-packages/ncTelegram/ui_msgwidget.py", line 60, in get_history
    self.print_msg(msg)
  File "/usr/local/lib/python3.5/dist-packages/ncTelegram/ui_msgwidget.py", line 194, in print_msg
    self.focus_position = self.pos
  File "/usr/local/lib/python3.5/dist-packages/urwid/listbox.py", line 578, in set_focus
    raise IndexError("Can't set focus, ListBox is empty")
IndexError: Can't set focus, ListBox is empty
Error in atexit._run_exitfuncs:
Traceback (most recent call last):
  File "/usr/lib/python3.5/subprocess.py", line 1072, in communicate
    stdout, stderr = self._communicate(input, endtime, timeout)
  File "/usr/lib/python3.5/subprocess.py", line 1700, in _communicate
    input_view = memoryview(self._input)
TypeError: memoryview: a bytes-like object is required, not 'str'
read: Connection reset by peer
```
Forcing a downgrade to urwid==1.3 opens correctly

**Comment by NiTHIN-K** [comment]
Exact same error here

**Comment by madc0ww** [comment]
Same error here

**Comment by amanusk** [comment]
The best fix for this is to install in a virtualenv, where urwid 1.3 can be used. 
It seems this repository has not been maintained in a while, so a fix is probably not coming.

**Comment by madc0ww** [comment]
Indeed. I'm gonna fork it and check if it is a bug, or if the failure happens due to a change on *urwid* API.

**Comment by Nanoseb** [comment]
Hello all,
As @amanusk wrote, I don't have time to maintain this anymore. But I will be happy to merge pr.
@amanusk if you want to describe how you circumvent this in the readme file feel free to do so. 
Or if someone wants to fix the code to make it compatible with urwid 2.0 that would be great too.
Anyway, sorry for not maintaining it anymore and thanks for reporting this.

**Comment by Mikepicker** [comment]
Fixed here #40

**Comment by Nanoseb** [comment]
ah yes, forgot to close that one, thx
